Q/qwtw/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "qwtw"
+version = v"2.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ig-or/qwtw.git", "b9451c91004de517e97429cd762caaad206f6014")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd qwtw
+mkdir build
+cd build
+export LD_LIBRARY_PATH=/workspace/srcdir/qwtw/build:$LD_LIBRARY_PATH
+ln -s /lib64/libc.so.6 libc.so
```

Comment:
These lines are useless at best, harmful at worst
```suggestion
```

Reply:
Build with  BinaryBuilder on my side told me that libc.so was not found, that is why I added this (after noticing similar things in other scripts).

Reply:
> similar things in other scripts

Which ones?

Reply:
> Build with BinaryBuilder on my side told me that libc.so was not found

This looks to me a problem with the executable being run, creating that symlink is a bad cure

Reply:
Qt itself, line 36. 
This is from Qt/build_tarballs.jl:           ln -s /lib64/libc.so.6 /lib64/libc.so

Reply:
@barche :flushed:

Reply:
See https://github.com/JuliaPackaging/Yggdrasil/commit/e7967da3858f44d846d49efb66395d7228dd371d#commitcomment-44407292

Reply:
@barche, any advice from your side about ways to handle this `ln -s /lib64/libc.so.6 /lib64/libc.so ` for qwtw? Can I dare to put it back in qwtw script?

Reply:
I think adding `apk add g++` is the less shocking way, still not entirely sure why this is needed though.